### ISSUE TITLE
wrong way in adding flatten layer

### ISSUE THREAD

**Issue by bacterous** [dingchenwei/googLeNet#3]
In googLeNet.py line 49:
``linear = Dense(units=1000, activation="softmax", `kernel_regularizer=l2(0.01))(keras.layers.core.Flatten(drop1))`` 
while  it should be:
``linear = Dense(units=1000, activation="softmax", kernel_regularizer=l2(0.01))(keras.layers.core.Flatten()(drop1))``

**Comment by ghost** [comment]
hello, have you ever solved the problem? Could you please give me some advice? thank you
